Cargo.toml

```diff
@@ -23,3 +23,4 @@ chrono = "0.4"
 parse_duration = "2.1.1"
 atty = "0.2"
 base64 = "0.21.0"
+tempdir = "0.3.7"
```

Comment:
Since this is only used during development, let's add a new section called (`[dev-dependencies]`](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies) and add `tempdir` to that section instead